### PR TITLE
[APPLICATIONS] [SCREENSAVERS] Add Portuguese (PT) translation

### PR DESCRIPTION
**Pull request by zecarlos1957**
[APPLICATIONS] [SCREENSAVERS] Add Portuguese (PT) translation

[SYSSETUP] Update Portuguese translation

### CODE REVIEW

File: base/applications/screensavers/3dtext/lang/pt-PT.rc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/* Translation to Portuguese (PT) by zecarlos1957@hotmail.com 10-09-2020 */
+LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
```
Comment (KyleKatarn31) [inline base/applications/screensavers/3dtext/lang/pt-PT.rc:2]: Why keeping neutral here ?

File: base/applications/screensavers/logon/lang/pt-PT.rc [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/* Translation to Portuguese (PT) by zecarlos1957@hotmail.com 10-09-2020*/
+
+LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
```
Comment (KyleKatarn31) [inline base/applications/screensavers/logon/lang/pt-PT.rc:3]: Keep neutral?

File: base/applications/screensavers/3dtext/lang/pt-PT.rc (hunk 1 above)
Comment (binarymaster) [reply base/applications/screensavers/3dtext/lang/pt-PT.rc:2]: Do you think it's not correct? If yes, provide some details.

File: base/applications/screensavers/logon/lang/pt-PT.rc [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+STRINGTABLE
+BEGIN
+    IDS_DESCRIPTION "Protecção de ecrã Início"
+    IDS_TEXT "Esta protecção de ecrã não é configuravel"
```
Comment (HBelusca) [inline base/applications/screensavers/logon/lang/pt-PT.rc:8]:
```suggestion
    IDS_TEXT "Esta protecção de ecrã não é configuravel."
```

File: dll/win32/syssetup/lang/pt-PT.rc [hunk 4]
```diff
@@ -23,16 +23,16 @@ BEGIN
...
-CAPTION "ReactOS Setup"
+CAPTION "Instalação do ReactOS"
 STYLE DS_MODALFRAME | DS_SHELLFONT | WS_POPUPWINDOW | WS_CAPTION | WS_VISIBLE
 FONT 8, "MS Shell Dlg"
 BEGIN
     ICON "", IDC_PRODUCT_ICON, 5, 5, 20, 20
-    LTEXT "Please choose a product option:", IDC_STATIC, 35, 7, 230, 12
-    LTEXT "Product &Options:", IDC_STATIC, 5, 32, 85, 10
+    LTEXT "Por favor escolha uma opção de produto:", IDC_STATIC, 35, 7, 230, 12
+    LTEXT "Opções de produto:", IDC_STATIC, 5, 32, 85, 10
```
Comment (HBelusca) [inline dll/win32/syssetup/lang/pt-PT.rc:32]:
```suggestion
    LTEXT "&Opções de produto:", IDC_STATIC, 5, 32, 85, 10
```

File: base/applications/screensavers/3dtext/lang/pt-PT.rc (hunk 1 above)
Comment (zecarlos1957) [reply base/applications/screensavers/3dtext/lang/pt-PT.rc:2]: https://reactos.org/wiki/Sublang

File: dll/win32/syssetup/lang/pt-PT.rc (hunk 4 above)
Comment (zecarlos1957) [reply dll/win32/syssetup/lang/pt-PT.rc]: Thanks

**Review by HBelusca — APPROVED** [review]

**Pull request merged**